from0k2bp.tex

```diff
@@ -1566,9 +1566,9 @@
 
 But for the case $600 = 10 \times 10 \times 6$ - we first ``chunk'' in 10s, then
 again in 10s, leaving only 6 components for the final step. That
-requires revealing $2\times 10-1 = 19$ commitments at each of the two reducing
+requires revealing $2\times (10 - 2) = 18$ commitments at each of the two reducing
```

Comment:
I guess we can clarify this and the next bit of arithmetic via #19 but also 2 x (10-2) is not 18 :)

Reply:
My true (in)ability is exposed!